configuration.rst

```diff
@@ -872,6 +872,27 @@ the env files ending in ``.local`` (``.env.local`` and ``.env.<environment>.loca
 **should not be committed** because only you will use them. In fact, the
 ``.gitignore`` file that comes with Symfony prevents them from being committed.
 
+Overriding Environment Values Defined By The System
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You may need to override an environment value defined by the system. This can
+be done thanks to the :class:`Symfony\\Component\\Dotenv\\Dotenv` class. The
+:method:`Symfony\\Component\\Dotenv\\Dotenv::loadEnv`,
+:method:`Symfony\\Component\\Dotenv\\Dotenv::bootEnv`, and
+:method:`Symfony\\Component\\Dotenv\\Dotenv::populate` methods accept an optional
+``overrideExistingVars`` parameter that allows you to override existing environment
+variables set by the system::
+
+    use Symfony\Component\Dotenv\Dotenv;
+
+    $dotenv = new Dotenv();
+    $dotenv->loadEnv(__DIR__.'/.env', null, 'dev', ['test'], true);
```

Comment:
I would use named arguments here and only add the required one + the overrideExistingVars parameter

Reply:
Yes but 5.4 minimal req is PHP 7.2. Shouldn't the example be compatible with the lowest possible PHP version ?

Reply:
Oh indeed, but we can do it after upmerge

Reply:
I'll provide the PR :slightly_smiling_face: